applet/src/lib.rs

```diff
@@ -120,7 +120,7 @@ impl CosmicAppletHelper {
                 resizable: None,
                 ..Default::default()
             }),
-            ..cosmic::settings_with_flags(flags)
+            ..Settings::with_flags(flags)
```

Comment:
We probably want to add `default_font: cosmic::font::FONT` and `default_text_size: 18.0`, since those were set in `cosmic::settings_with_flags`.